docs/cristin-proxy-swagger.yaml

```diff
@@ -1999,33 +1998,33 @@ components:
             description: 'Date of approval'
           authority:
             type: string
-            description: 'Approval authority. Only certain combinations of authority and applicationCode are allowed. REK: Regional committees for medical and health research ethics (en), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nb), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nn), application_code: ETICHAPPR or BIOBANK; NARA: The Norwegian Animal Research Authority (en), Forsøksdyrutvalget (nb), Forsøksdyrutvalet (nn), application_code: TESTANIMAL; NDPA: The Norwegian Data Protection Authority (en), Datatilsynet (nb), Datatilsynet (nn), application_code: SENSINFO; NMA: Norwegian Medicines Agency (en), Statens legemiddelverk (nb), Statens legemiddelverk (nn), application_code: DRUGTRIAL; DIRHEALTH: Norwegian Directorate of Health (en), Helsedirektoratet (nb), Helsedirektoratet (nn), application_code: BIOTECHN or MEDEQUIP.'
+            description: 'Approval authority. Only certain combinations of authority and applicationCode are allowed. Rek: Regional committees for medical and health research ethics (en), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (Rek) (nb), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (Rek) (nn), applicationCode: EtichAppr or Biobank; Nara: The Norwegian Animal Research Authority (en), Forsøksdyrutvalget (nb), Forsøksdyrutvalet (nn), applicationCode: TestAnimal; Ndpa: The Norwegian Data Protection Authority (en), Datatilsynet (nb), Datatilsynet (nn), applicationCode: SensInfo; Nma: Norwegian Medicines Agency (en), Statens legemiddelverk (nb), Statens legemiddelverk (nn), applicationCode: DrugTrial; Dirhealth: Norwegian Directorate of Health (en), Helsedirektoratet (nb), Helsedirektoratet (nn), applicationCode: BioTechN or MedEquip.'
             enum:
-              - REK
-              - NARA
-              - NDFA
-              - NMA
-              - DIRHEALTH
+              - Rek
+              - Nara
+              - Ndfa
+              - Nma
+              - Dirhealth
```

Comment:
We should probably provide actual names rather than abbreviations since I don't know what NARA, NOFA, NMA is. Why is DirHealth in English?

Reply:
What should we call them then? Use the full name?